.ci/images_matrix.json

```diff
@@ -1,5 +1,5 @@
 {
-    "android_api": [ 28, 29, 30],
+    "android_api": [ 28, 29, 30, 32],
```

Comment:
Already in the future @ferrandp  ? You're testing the new beta ? 😛 

You might need the version 31 too ?